py4cast/datasets/poesy.py

```diff
@@ -902,19 +899,20 @@ def from_json(
                 )
                 param_list.append(param)
 
-        inference_period = Period(**conf["periods"]["test"], name="infer")
-        ds = InferPoesyDataset(
-            grid,
-            inference_period,
-            param_list,
-            PoesySettings(
-                members=members,
-                term=term,
-                num_input_steps=num_input_steps,
-                num_output_steps=0,
-                num_inference_pred_steps=config_override["num_inference_pred_steps"],
-            ),
-        )
+            inference_period = Period(**conf["periods"]["test"], name="infer")
```

Comment:
Here the code as been indented, pushing the dataset instantiation in the for loop (cf my other comment)